src/docs/product/alerts-notifications/alert-settings.mdx

```diff
@@ -8,7 +8,9 @@ Configure alerts to your team's needs with alert rules and settings. Create new
 
 ## Rules
 
-The “Alert Rules” tab displays your alert rules, along with their current status, project, team, and date created. If you have [Performance Monitoring](/product/performance/) configured for your organization, the **Alerts** page also displays a “History” tab where you can find a list of triggered alerts, along with visual graphs, associated projects, and the last time the alert was triggered. In the “History” tab, the default view displays unresolved alerts.
+The “Alert Rules” tab displays your alert rules, along with their current status, project, team, and creation date. The list is filtered so that only alerts associated with the teams you're a member of are displayed. You can change this using the "Filter" button. The name of the button changes to reflect the number of filters applied, so when you first navigate to the page, it's labeled "2 Active Filters".
```

Comment:
Is the last sentence an example (if so, we should say "for example")?

Reply:
AFAIK that's the default number of filters for everyone (updated screenshot coming), but @iProgramStuff should confirm.

Reply:
It is the same. It's still an example though, right?

Reply:
I'm trying to explain the default behaviour, mostly because I'm worried about folks coming to the page, expecting to see a button labeled "Filter" and not seeing that, but instead seeing a button that says "2 Active Filters." How does this work @iProgramStuff and @PeloWriter 

```suggestion
The “Alert Rules” tab displays your alert rules, along with their current status, project, team, and creation date. The list is filtered so that only alerts associated with the teams you're a member of, as well as ones that aren't associated with any team are displayed. You can change this using the "Filter" button. The name of the button changes to reflect the number of filters applied, so when the default filters are selected, it's labeled "2 Active Filters".
```